insolar/genesis.go

```diff
@@ -16,21 +16,45 @@
 
 package insolar
 
+import "strconv"
+
 const (
 	// GenesisNameRootDomain is the name of root domain contract for genesis record.
 	GenesisNameRootDomain = "rootdomain"
 	// GenesisNameNodeDomain is the name of node domain contract for genesis record.
 	GenesisNameNodeDomain = "nodedomain"
 	// GenesisNameNodeRecord is the name of node contract for genesis record.
 	GenesisNameNodeRecord = "noderecord"
-	// GenesisNameRootMember is the name of root member contract for genesis record.
-	GenesisNameRootMember = "member"
-	// GenesisNameRootWallet is the name of wallet contract for genesis record.
-	GenesisNameRootWallet = "wallet"
-	// GenesisNameAllowance is the name of allowance contract for genesis record.
-	GenesisNameAllowance = "allowance"
+	// GenesisNameMember is the name of member contract for genesis record.
+	GenesisNameMember = "member"
+	// GenesisNameWallet is the name of wallet contract for genesis record.
+	GenesisNameWallet = "wallet"
+	// GenesisNameDeposit is the name of deposit contract for genesis record.
+	GenesisNameDeposit = "deposit"
+	// GenesisNameTariff is the name of tariff contract for genesis record.
+	GenesisNameTariff = "tariff"
+	// GenesisNameCostCenter is the name of cost center contract for genesis record.
+	GenesisNameCostCenter = "costcenter"
+
+	GenesisNameRootMember           = "root" + GenesisNameMember
+	GenesisNameRootWallet           = "root" + GenesisNameWallet
+	GenesisNameMigrationAdminMember = "migrationadmin" + GenesisNameMember
+	GenesisNameMigrationWallet      = "migration" + GenesisNameWallet
+	GenesisNameCommissionWallet     = "commission" + GenesisNameWallet
+
+	GenesisNameStandardTariff = "standard" + GenesisNameTariff
+
+	GenesisAmountMigrationDamonMembers       = 10
+	GenesisAmountActiveMigrationDamonMembers = 3
```

Comment:
what is "Damon"?

Reply:
everywhere

Reply:
It is the name of Oracle

Reply:
In docs its "Migration D**ae**mon", not D**a**mon